app/Database/DataType.hs

```diff
@@ -17,3 +17,17 @@ derivePersistField "ShapeType"
 
 instance ToJSON ShapeType
 instance FromJSON ShapeType
+
+-- data AlignType = Begin | Start | Middle | End
```

Comment:
Please remove this from this branch (of course this is a good change to make, but should be done on a separate branch).